index.bs

```diff
@@ -44,6 +44,11 @@ spec: sensors; urlPrefix: https://w3c.github.io/sensors/#
 spec: manifest; urlPrefix: https://w3c.github.io/manifest/#
     type: dfn
         text: install; url: dfn-install
+spec: feature-policy; urlPrefix: https://wicg.github.io/feature-policy/#
+    type: dfn
+        text: policy-controlled feature
```

Comment:
Feature Policy has gotten into Shepherd since you wrote this, so you can undo this addition.